### PR TITLE
E2e testing suitev1

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
**Description of the Pull Request (PR):**

Feedback on #2591 is addressed here, new PR to avoid messy merge conflicts 

**This fixes or addresses the following GitHub issues:**

- Fixes #

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -178,7 +179,7 @@ workflows:
       - unit_tests:
           requires:
             - get_source
-      - integration_tests:
+      - e2e_test:
```
Comment (jscook2345) [inline .circleci/config.yml]: `e2e_tests` instead of `e2e_test`

File: .circleci/config.yml [hunk 2]
```diff
@@ -117,7 +117,7 @@ jobs:
             cd $HOME/go/src/github.com/sylabs/singularity
             make -j$(nproc) -C ./builddir unit-test
 
-  integration_tests:
+  e2e_test:
```
Comment (jscook2345) [inline .circleci/config.yml]: `e2e_tests` instead of `e2e_test`

File: .circleci/config.yml [hunk 3]
```diff
@@ -128,6 +128,7 @@ jobs:
             echo 'export GOPATH=$HOME/go' >> $BASH_ENV
             echo 'export GOROOT=/usr/local/go' >> $BASH_ENV
             echo 'export GOBIN=$HOME/go/bin' >> $BASH_ENV
+            echo 'export SINGULARITY_E2E=true' >> $BASH_ENV
```
Comment (jscook2345) [inline .circleci/config.yml]: 0 or 1 for a bash variable is standard for bool

**Review by jscook2345 — APPROVED** [review]
LGTM. We can start from here

**Review by tri-adam — APPROVED** [review]
Thanks @ArangoGutierrez. Think this is a good start. Let's get this draft merged so folks can start hacking at it.

**Pull request merged**